docs/reference/remap/format_timestamp.cue

```diff
@@ -0,0 +1,52 @@
+package metadata
+
+remap: functions: format_timestamp: {
+	arguments: [
+		{
+			name:        "value"
+			description: "The timestamp to format as text."
+			required:    true
+			type: ["timestamp"]
+		},
+		{
+			name:        "format"
+			description: "The format string"
+			required:    true
+			type: ["string"]
+		},
+	]
+	return: ["string"]
+	category: "text"
+	description: #"""
+			Formats a `timestamp` as a given string.
+			The format string used is specified by the [Chrono library](https://docs.rs/chrono/0.4.19/chrono/format/strftime/index.html).
```

Comment:
Is it ok to just reference the chrono docs here?

Reply:
This seems fine to me. We'll probably reference the regex crate documentation somewhere as well to explain its syntax.